Cannot run jupyter kernel using python extension

I upgraded to the latest code-server today and now the jupyter interactive functionality of the python extension has stopped working. This happens both when I did this using a remote linux machine (docker based on the pytorch 1.9 image), AND when installing code-server on my local macbook. When opening a jupyter notebook a popup with the following information appears:

```Command 'Jupyter: Create New Blank Notebook' resulted in an error (command 'jupyter.createnewnotebook' not found)```


## OS/Web Information

- Web Browser: brave
- Local OS: macOs
- Remote OS: macos
- Remote Architecture: 
- `code-server --version`: 3.11.0 4e8cd09ef0412dfc7b148b7639a692e20e4fd6dd

## Steps to Reproduce

On macos (m1):
1. install code-server using install script
2. install python extension (ms-python.python v2021.7.1060902895). This will also install a jupyter extension (ms-toolsai.jupyter v2021.6.832593372)
![image](https://user-images.githubusercontent.com/7136076/127607791-eba77701-7cef-4faa-ad44-21f548d3a0e2.png)

3. Try to open a new jupyter notebook (cmd+shift+p "Jupyter: Create new Blank Notebook")
![image](https://user-images.githubusercontent.com/7136076/127607905-f2119e1c-9fe4-4bbf-8854-435f88726c3a.png)
![image](https://user-images.githubusercontent.com/7136076/127608002-f08ba97f-e1c3-4cf2-a415-79fc74db2bb5.png)

The developer console has this error (which is too far in the deep for me to understand):
![image](https://user-images.githubusercontent.com/7136076/127608358-3213bd24-0950-4e49-b93b-06c333f099d4.png)


## Expected
A jupyter notebook starts up
## Actual
Error message 
```Command 'Jupyter: Create New Blank Notebook' resulted in an error (command 'jupyter.createnewnotebook' not found)```

I'm going to start working on this issue and use this comment as a place to document my notes. I will update it as I find more information so as to not spam the issue.

## Try to Reproduce Issue Locally

The first question is to see if this is still an issue. Here's how I'm going to try to reproduce it:
1. using code-server 3.12.0, run `code-server --extensions-dir cs-3874` (`cs-3874` is the name of an empty directory. I do this so I start code-sever with no extensions installed, a clean slate.)
2. Download the Python extension from Open VSX and manually install with `.vsix` file. 
3. Reload page in browser
4. Verify both Python and Jupyter are installed: `v2021.9.1230869389` and `v2021.8.12` respectively
5. Command Palette > Jupyter: Create New Blank Notebook

### Expected

A jupyter notebook starts up

### Actual

Command fails

![image](https://user-images.githubusercontent.com/3806031/134258696-5a526b07-b9e4-4c13-b655-da45d7c6b6b3.png)

✅ Confirmed. This issue still exists in code-server 3.12.0

### Notes

Based on the error in the console and the stack trace, it looks like we're missing a module in our fork of VS Code (you can tell because the stack points to `vendor/modules/code-oss-dev` which is where our fork of VS Code lives). 

```javascript
mainThreadExtensionService.ts:95 Activating extension 'ms-toolsai.jupyter' failed: Cannot find module '/Users/jp/Dev/repros/cs-3874/ms-toolsai.jupyter-2021.8.12/out/client/extension'
Require stack:
- /usr/local/Cellar/code-server/3.12.0/libexec/vendor/modules/code-oss-dev/out/vs/loader.js
- /usr/local/Cellar/code-server/3.12.0/libexec/vendor/modules/code-oss-dev/out/bootstrap-amd.js
- /usr/local/Cellar/code-server/3.12.0/libexec/vendor/modules/code-oss-dev/out/bootstrap-fork.js.
```

Well actually, It looks like it can't find the module in `cs-3874/ms-toolsai.jupyter-2021.8.12/out/client/extension` 🤔

Let's go there and take a look...

--- a/ci/build/build-release.sh
+++ b/ci/build/build-release.sh
@@ -49,7 +49,7 @@ bundle_code_server() {
   {
     "commit": "$(git rev-parse HEAD)",
     "scripts": {
-      "postinstall": "./postinstall.sh"
+      "postinstall": "sh ./postinstall.sh"
     }
   }
 EOF

--- a/ci/build/build-release.sh
+++ b/ci/build/build-release.sh
@@ -49,7 +49,7 @@ bundle_code_server() {
   {
     "commit": "$(git rev-parse HEAD)",
     "scripts": {
-      "postinstall": "bash ./postinstall.sh"
+      "postinstall": "./postinstall.sh"
     }
   }
 EOF

